cypress/e2e/navigation.cy.ts

```diff
@@ -72,4 +72,36 @@ describe('Navigation Test', () => {
     profilePage.verifySchoolDetails(schoolDetails);
     cy.logout();
   });
+
+  it('Should navigate to enrolled contests and verify filter', () => {
+    const userLoginOptions = loginPage.registerMultipleUsers(1);
+    cy.login(userLoginOptions[0]);
+
+    cy.get('[data-nav-user]').should('be.visible');
+    cy.get('[data-nav-user]').click();
+    cy.get('[data-nav-user-contests]').should('be.visible');
+
+    cy.get('[data-nav-user-contests]').first().click();
```

Comment:
En las pruebas de Cypress (y en general en todas las pruebas)  evitamos el uso de selectores posicionales. 

¿A qué me refiero? Que en un dado caso que se agreguen mas elementos que coincidan con el selector y estos aparezcan antes del que se está haciendo referencia, esta prueba va a fallar.

Así que estuve revisando y `data-nav-user-contests` se está usando en dos opciones del menú que te redirigen a Concursos. 

Puedes modificar en el componente de Navbar el `data-attribute` de los `contests enrolled` por `data-nav-user-contests-enrolled` y acá puedes usar ese nuevo `data-attribute` y quitar el método `first()`:
 ```suggestion
     cy.get('[data-nav-user-contests-enrolled]').click();
```
no olvides también cambiar el selector de la línea 82